### PR TITLE
Add ARM64 in goreleaser and Travis-CI

### LINKED ISSUE #1486: Require ARM64 docker images and binaries to add ARM64 job in travis
Hi,

I am working on adding an ARM64 job in Travis. ARM64 job is failing as docker image **dragonflyoss/nginx:apline** is not available for arm64.

Please check https://travis-ci.org/github/odidev/Dragonfly/jobs/729522338 for the same.
`standard_init_linux.go:211: exec user process caused "exec format error"`

I have built ARM64 binaries as well in ARM64 platform using below steps:

- make build-supernode
- make build-client

```
$ ls bin/linux_arm64/
dfdaemon  dfget  supernode

$ file bin/linux_arm64/*
bin/linux_arm64/dfdaemon:  ELF 64-bit LSB executable, ARM aarch64, version 1 (SYSV), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, Go BuildID=siPFbi0V2yoHRNpBwRLL/8WMZ7AX1Wrj00q8trB0H/9PArBl2vblLhnnrx492P/P42fsllta8DvvWF91QuE, not stripped
bin/linux_arm64/dfget:     ELF 64-bit LSB executable, ARM aarch64, version 1 (SYSV), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, Go BuildID=hb3d8LxOV-KvorpHfdYY/NoQlyPdkzqkKlTBD93-z/Yz-IEre11oQ46DRFbnJ_/KxUsK3wSOj6y-2PXa1_Y, not stripped
bin/linux_arm64/supernode: ELF 64-bit LSB executable, ARM aarch64, version 1 (SYSV), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, Go BuildID=hftaDpkvPpAgjtyMljaw/yYqy5u0bD6mWpvkjtYdH/Qr5kyabH72ZD-_cRiU73/JODvczSLvRkVZ4O86EnR, not stripped

```
It will be very helpful if binaries and images are released for ARM64 platform. Please let me know if any help is required from my end. I am very much happy to help.

### PR DESCRIPTION
**Pull request by odidev**
- Updated the .goreleaser.yml file to generate linux/arm64 binaries.
- Added arm64 job in Travis-CI.
- Replaced dragonflyoss/nginx:apline with nginx:1.19.2-alpine image for supernode image.

Resolves #1486.

### CODE REVIEW

**Comment by jim3ma** [comment]
LGTM

**Comment by zhouhaibing089** [comment]
LGTM

File: .goreleaser.yml [hunk 1]
```diff
@@ -13,9 +13,13 @@ builds:
     main: cmd/supernode/main.go
     goarch:
       - amd64
+      - arm64
     goos:
       - linux
       - darwin
+    ignore:
+      - goos: darwin
+        goarch: arm64
```
Comment (zhouhaibing089) [inline .goreleaser.yml:22]: Could you help me understand why this needs to be excluded?

Comment (odidev) [reply .goreleaser.yml:22]: It is trying to get Darwin OS with ARM64 architecture, which is not available. So, I have added in ignore step to exclude Darwin with ARM64 build.

**Comment by odidev** [comment]
Thanks for merging. May I know when are you planning a release with arm64 binaries?

**Pull request merged**